connectors/github/element-templates/github-connector.json

```diff
@@ -377,6 +381,29 @@
         "name": "operationType"
       }
     },
+    {
+      "label": "Operation type",
```

Comment:
Perhaps I'm missing something, but did you consider including this content operation as part of the existing repositories OperationType?

It seems to be part of the [GitHub Repositories API](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28), so I just wanted to double-check 😄

Reply:
I did consider it but I wasn't sure how your team wants to group the APIs given the nested structure of GitHub's repository API.

I find it a bit confusing that the endpoints for direct repository operations live below "Repositories > Repositories", e.g. "[List organization repositories](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-organization-repositories)" while this operations is situated below "Repositories > Content > [Get repository content](https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28#get-repository-content)". Would you expect this operation to be part of the "Repositories" operation type?

Reply:
I personally would expect the "Get repository content" to be under "Repositories", because "Repository" is a root-level concept in GitHub's API. I also think "Content" as an operation group is a little too vague, since it could easily be confused for "pull request content" or "release content", etc. to someone scanning the dropdowns quickly.

@Oleksiivanov @sbuettner what are your thoughts here? (I ask since you both better understand customer's expectations around Connectors)

Reply:
I share your perspective on this @ev-codes . @wollefitz would that work for you?

Reply:
> @wollefitz would that work for you?

Sure, implemented via https://github.com/camunda/connectors/pull/2186/commits/edcd47177d1145a4a9caf4c54a803a76f51376a1.